emr-etl-runner/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM jruby:9.1
+MAINTAINER Josh Cox <josh 'at' webhosting.coop>
+
+ENV PLAIN_OF_JARS=20170705 \
+LANG=en_US.UTF-8
+
+RUN apt-get update \
+&& apt-get install -yqq gettext-base mlocate \
```

Comment:
I would remove anything StorageLoader related. The running of StorageLoader is moving into EMR in R90, and that won't allow running through Docker.

Eventually the heir-of-StorageLoader will be runnable as part of the real-time pipeline, but that is a way off and if/how this can be Dockerized is still to be seen.